dvc/command/plots.py

```diff
@@ -198,6 +210,16 @@ def add_parser(subparsers, parent_parser):
     )
     plots_modify_parser.set_defaults(func=CmdPlotsModify)
 
+    SHOW_HELP = "Dump plots templates."
```

Comment:
```suggestion
    SHOW_HELP = "Write plots template JSON schemas to files."
```

Not sure if this wording is great, but a bit more description would be nice. cc @jorgeorpinel

Reply:
I skimmed through #6389 and didn't really get why we're dumping plot template files. Will they no longer be included in `.dvc/` ? (Compatibility-breaking change BTW?)

Reply:
UPDATE: Duh it's explained in the PR description... OK ⌛

Reply:
Idk if `dump` is a good name in the first place. Sounds like what we're really doing is generating default templates. Why not `dvc plots templates`?

```suggestion
    SHOW_HELP = "Write built-in plots templates to a directory."
```